portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -122,12 +123,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
  
     String nickname = request.getParameter(SENDER_PARAMETER);
     String postTag = request.getParameter(TAG_PARAMETER);
+    String parentIDString = request.getParameter(PARENTID_PARAMETER);
+    long parentID = Long.parseLong(parentIDString);
```

Comment:
You would probably want to handle the parse exception here if |parentIDString| is not a valid long integer.

Reply:
For better error messages to the client side.